### ISSUE TITLE
how to publish CSE software suggestion

### ISSUE THREAD

**Issue by danielskatz** [betterscientificsoftware/betterscientificsoftware.github.io#163]
There is already a larger list (https://www.software.ac.uk/which-journals-should-i-publish-my-software) that perhaps should just be referred to in https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/CuratedContent/HowToPublishCseSw.md

Also, I wonder if this page should include discussion about the GitHub -> Zenodo and figshare links, since those are actually about publishing software, rather than publishing papers about software.

**Comment by ChrisRackauckas** [comment]
Yes, I think there should really be a good up to date article on this because when I was looking to publish on my software I found lots of old information.

**Comment by maherou** [comment]
This article has moved:  https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/CuratedContent/PublishingCseSw.md

@ibaned: Are you interested in revising your document?

**Comment by ibaned** [comment]
@maherou yes I'd be happy to.

@danielskatz thanks for the longer list! I've opened PR #247 to at least reference this list

@ChrisRackauckas are there other examples of "old information" that you think should also be updated as part of this PR?

**Comment by ChrisRackauckas** [comment]
Looks like https://www.software.ac.uk/which-journals-should-i-publish-my-software got a refresh since I last saw it. I think the new 

https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/CuratedContent/PublishingCseSw.md

is much better though: more concise and hits the big players. However, I wouldn't put Zenodo with journals since it's really a different thing. It and figshare should be mentioned separately.

**Comment by ibaned** [comment]
Should JOSS also be in that category? There is this gray area where one just obtains a DOI for the software without having to write a full paper, and I think JOSS is closer to that than a full-paper journal.

**Comment by ibaned** [comment]
I separated the list, and for now kept JOSS with the other "full journals".

**Comment by danielskatz** [comment]
JOSS is a journal - authors do have to write a paper.  (though it is a short paper)

**Comment by ibaned** [comment]
@danielskatz  Oh, okay. then I think the lists are correct as they are in the pull request.

@maherou as far as I can tell, the pull request is ready to go per the comments of @danielskatz and @ChrisRackauckas .
